test-helpers/src/mpc_network.rs

```diff
@@ -0,0 +1,101 @@
+//! Groups helpers for setting up and managing an MPC network
+pub mod field;
+pub mod mocks;
+
+use std::net::SocketAddr;
+
+use dns_lookup::lookup_host;
+use futures::{executor::block_on, Future};
+use mpc_stark::{
+    network::{PartyId, QuicTwoPartyNet},
+    MpcFabric, PARTY0, PARTY1,
+};
+use tokio::io::duplex;
+
+use crate::mpc_network::mocks::MockNetwork;
+
+use self::mocks::PartyIDBeaverSource;
+
+/// The size of MPC network to allocate by default
+pub const DEFAULT_MPC_SIZE_HINT: usize = 1_000_000;
+
+// -----------
+// | Helpers |
+// -----------
+
+/// Sets up a basic MPC fabric between two parties using the QUIC transport and
+/// the beaver triplet source defined above
+pub fn setup_mpc_fabric(party_id: u64, local_port: u64, peer_port: u64, docker: bool) -> MpcFabric {
```

Comment:
this method was copied from the analogous method in `integration-helpers` crate, which was deleted in favor of this crate